src/bin/www.ts

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
```

Comment:
should be added to git?

Reply:
this line or this file?
this is the file that runs when you do `npm start`
and the line its for unix-like system